ceph-dev-new-trigger/config/definitions/ceph-dev-new-trigger.yml

```diff
@@ -152,6 +152,26 @@
                     DISTROS=centos8
                     FLAVOR=crimson
                     ARCHS=x86_64
+      # Build only the `crimson` flavour, don't waste resources on the default one.
+      # Useful for the crimson's bug-hunt at Sepia
+      - conditional-step:
+          condition-kind: regex-match
+          regex: .*crimson-only.*
```

Comment:
might want to add another regex to the previous `conditional-step` , so the `condition-command` reads:
```sh
echo "${GIT_BRANCH}" | grep -v '\(luminous\|mimic\|nautilus\|octopus\|pacific\|crimson-only\)'
```
otherwise, the crimson-only branches would trigger the ceph-dev-new job 3 times. as it matches with 

- `grep -v '\(luminous\|mimic\|nautilus\|octopus\|pacific\)'` *and*
- `.*crimson-only.*`